WordPress/src/main/java/org/wordpress/android/ui/people/PersonDetailFragment.java

```diff
@@ -106,4 +157,104 @@ public void setPersonDetails(long personID, int localTableBlogID) {
         mLocalTableBlogID = localTableBlogID;
         refreshPersonDetails();
     }
+
+    // Checks current user's capabilities to decide whether she can change the role or not
+    @SuppressWarnings("deprecation")
```

Comment:
This comment is more like a personal preference so feel free to move passed it: I'd recommend factoring out the code that the `SuppressWarnings` is about (I assume it's the `setBackgroundDrawable()` call) into its own function and have the suppression apply to that one.
The idea is to avoid having suppressions on "biggish" functions which might hide potential bugs. I would suggest a `clearRoleContainerBackground()` function or something in this case. Cheers!